test/lib/mayaUsd/render/mayaToHydra/testMayaDefaultMaterial.py

```diff
@@ -45,15 +46,14 @@ def test_MayaDefaultMaterial(self):
             cmds.refresh()
             self.assertSnapshotClose("defaultMaterial" + ".png", self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)        
 
+    @unittest.skipUnless(platform.system() != "Darwin", 'This test is disabled on OSX as not all implicit surfaces are fully implemented on OSX.')
```

Comment:
Who will support them on OSX? If it's us, should we track via a ticket?

Reply:
I created https://github.com/PixarAnimationStudios/OpenUSD/issues/3305 in the OpenUSd github to see if there is already something logged.